inst/templates/ad_adpc.R

```diff
@@ -0,0 +1,535 @@
+# Name: ADPC
+#
+# Label: Pharmacokinetics Concentrations Analysis Dataset
+#
+# Description: Based on simulated data, create ADPC analysis dataset
+#   The dataset format is also suitable for Non-compartmental analysis (ADNCA)
+#
+# Input: pc, ex, vs, adsl
+library(admiral)
+library(dplyr)
+library(lubridate)
+library(stringr)
+library(xportr)
+
+library(admiral.test) # Contains example datasets from the CDISC pilot project or simulated
+
+# Read in prepared spec file for ADPC ----
+adpc_spec <- readxl::read_xlsx("inst/specs/adpc_spec.xlsx", sheet = "Variables") %>%
+  dplyr::rename(type = "Data Type") %>%
+  rlang::set_names(tolower) %>%
+  mutate(format = str_to_lower(format))
```

Comment:
Hey @jeffreyad.  I really want this to happen, but it does add a lot of dependencies for the user.  I will bring this up to the admiral core team on Monday/Wednesday to discuss.  But this will be a separate issue for making this update. 

Can you store this on box for us to use later and revert your changes.  After that I think we can start merging in all these PK/PC materials you have been working on.

Reply:
@bms63 Sure, will do.  Thanks